package.json

```diff
@@ -82,8 +83,12 @@
   },
   "license": "MIT",
   "main": "./dist/Transloadit.js",
+  "types": "./dist/Transloadit.d.ts",
```

Comment:
This change is redundant. TypeScript’s `Node10` module resolution sees the `main` field, then infers a value of `./dist/Transloadit.js` means the types are in `./dist/Transloadit.d.ts`.

```suggestion
```